src/index.css

```diff
@@ -79,10 +79,40 @@ small,
 }
 
 .main {
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+.section-container {
   display: grid;
   justify-content: center;
   align-items: center;
-  min-height: 100vh;
+  min-height: calc(100vh - 3rem);
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav h3 {
+  margin: 0.24rem 0;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.nav-link {
+  padding: 0.25rem;
+  margin: auto 0.5rem;
+}
+
+.center {
+  border-left: 1px solid grey;
+  border-right: 1px solid grey;
```

Comment:
- [ ] Kindly use what you've already learnt in [React basic styling](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md) lesson for styling your app as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md#:~:text=In%20this%20project%2C%20you%20should%20use%20React%20basic%20styling.) (inline styles like [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#:~:text=Method%201%3A%20Inline,CSS%20rules%3A), using different CSS files one for each component like [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#:~:text=Method%202%3A%20Importing,way%20as%20usual%3A), or using CSS modules like [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md#:~:text=Method%203%3A%20CSS,module.css%20extension%3A) ) instead of adding general style in the index.css file.

Reply:
@ShadyShawkat  what do you mean by use inline CSS? if I add inline CSS for the whole project it will be cluttered and unreadable . the requirement said to Use React basic styling (inline styles, importing CSS, and pre-processed CSS files). so this means I can import CSS files which I did in the project. i used inline CSS in the navlink.

Reply:
@ambrose-kibet, What you did isn't importing CSS. importing CSS means that you can write CSS in different files and import them individually in their respective components. What you did was write all of the CSS in the `index.css` file which is imported into the `index.js` by default.

Reply:
@ShadyShawkat  as per  this [article](https://css-tricks.com/different-ways-to-write-css-in-react/) in [react_basic_styling](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/react_basic_styling.md) which was provided as reading material I did nothing wrong